tests/test_transforms/test_decomposition/test_binseg_trend_transform.py

```diff
@@ -31,8 +31,8 @@ def test_binseg_run_with_custom_costs(example_tsds: TSDataset, custom_cost_class
     """Check that binseg trend works with different custom costs."""
     bs = BinsegTrendTransform(in_column="target", custom_cost=custom_cost_class())
     ts = deepcopy(example_tsds)
-    ts.fit_transform([bs])
-    ts.inverse_transform()
+    ts = bs.fit_transform(ts)
+    ts = bs.inverse_transform(ts)
     assert (ts.df == example_tsds.df).all().all()
```

Comment:
If you need to get df from dataset it is better to use `to_pandas` method, try to fix it where possible